prototype/water-extent-map-on-demand.ipynb

```diff
@@ -0,0 +1,341 @@
+    "from IPython.display import display\n",
+    "\n",
+    "def wkt_input():\n",
+    "    wkt = widgets.Textarea(\n",
+    "        placeholder='WKT of search area',\n",
+    "        value='POLYGON((-91.185 36.6763,-86.825 36.6763,-86.825 38.9176,-91.185 38.9176,-91.185 36.6763))',\n",
+    "        layout=widgets.Layout(width='100%'),\n",
+    "    )\n",
+    "    display(wkt)\n",
+    "    return wkt\n",
+    "\n",
+    "def file_ids_input():\n",
+    "    file_ids = widgets.Textarea(\n",
+    "        placeholder='copy-paste Sentinel-1 granule names or file ids here (One granule or id per line)',\n",
+    "        layout=widgets.Layout(width='100%', height='12em'),\n",
+    "    )\n",
+    "    display(file_ids)\n",
+    "    return file_ids"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Search for Sentinel-1 scenes to process\n",
+    "\n",
+    "You can search for Sentinel-1 scenes with either [ASF Search -- Vertex](https://search.asf.alaska.edu/#/) or the\n",
+    "[asf_search](https://github.com/asfadmin/Discovery-asf_search) Python package. Vertex provides an interactive,\n",
+    "feature rich experience, while `asf_search` allows searching programmatically and mirrors the vertex interface\n",
+    "as best it can. Section 1.1 describes using Vertex and Section 1.2 describes using `asf_search`.\n",
+    "\n",
+    "*Note: only 1.1 or 1.2 needs to be executed to run this notebook.*\n",
+    "\n",
+    "### 1.1 Search for  Sentinel-1 scenes in Vertex\n",
+    "\n",
+    "Requesting water map products from the custom HyP3-watermap deployment looks very similar to\n",
+    "[requesting On Demand RTC products](https://storymaps.arcgis.com/stories/2ead3222d2294d1fae1d11d3f98d7c35),\n",
+    "**except** instead of adding scenes to your On Demand queue, you'll:\n",
+    "1. add the scenes to your Downloads cart\n",
+    "   ![add to cart](https://user-images.githubusercontent.com/7882693/122344682-85acc800-cef3-11eb-8337-5a356b722c12.png)\n",
+    "\n",
+    "\n",
+    "2. open the Downloads Cart and select \"Copy File Ids\", and\n",
+    "   ![image](https://user-images.githubusercontent.com/7882693/122345160-04096a00-cef4-11eb-8c27-892329293e4d.png)\n",
+    "\n",
+    "\n",
+    "3. paste the file ids into the text area that will appear below the next cell.\n",
+    "\n",
+    "**Note:** Water maps currently require the Sentinel-1 source granules to be SLCs (preferred) or High-Res GRDs,\n",
+    "acquired using the IW beam mode, with both VV and VH polarizations. You can use the\n",
+    "[example search](https://search.asf.alaska.edu/#/?beamModes=IW&polarizations=VV%2BVH&productTypes=SLC&zoom=6.190&center=-91.993,33.963&polygon=POLYGON((-91.185%2036.6763,-86.825%2036.6763,-86.825%2038.9176,-91.185%2038.9176,-91.185%2036.6763))&start=2021-05-30T00:00:00Z&resultsLoaded=true&granule=S1A_IW_GRDH_1SDV_20210607T234810_20210607T234835_038241_04834F_4BB6-GRD_HD&end=2021-06-07T23:59:59Z)\n",
```

Comment:
Might need to escape the `()` characters in the URL, they mess with rendering the link in github, haven't checked in jupyter

Reply:
Messes with the linting in Pycharm too but in a rendered markdown view it seems work fine (jupyter and pycharm)

![image](https://user-images.githubusercontent.com/7882693/122485132-a8d28880-cf82-11eb-8ae4-293ea69ff6d3.png)